content/react/react.md

```diff
@@ -9,8 +9,10 @@ CatalogContent:
 
 <link rel="canonical" href="https://www.codecademy.com/resources/blog/what-is-react/" />
 
-[React](https://reactjs.org/) is a free and open-source front-end JavaScript library for building user interfaces or webpage components. It was released in 2013 and is currently maintained by Facebook along with a community of developers and companies.
+[**React**](https://reactjs.org/) is a free and [open source](https://www.codecademy.com/resources/docs/open-source) [JavaScript](https://www.codecademy.com/resources/docs/javascript) library for building user interfaces or [front-end](https://www.codecademy.com/resources/docs/general/front-end) webpage components such as forms along wiht fields like text areas and buttons.
```

Comment:
~~[open source]~~ [open-source]

...webpage components such as forms along ~~wiht~~ with fields ~~like~~ such as areas and buttons.